pom.xml

```diff
@@ -42,7 +42,7 @@
         <compiler-plugin-version>3.8.0</compiler-plugin-version>
         <okhttp3-version>3.14.9</okhttp3-version>
         <logback-version>1.2.3</logback-version>
-        <guava-version>27.1-android</guava-version>
```

Comment:
🤔

Reply:
while the old version # refers to android, that version supported java 7+, but we no longer need java 7 support so I just moved us up to the latest.